src/extensions/default/CodeFolding/DefaultSettings.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * JSON values for default settings
+ * @author Patrick Oladimeji
+ * @date 8/23/14 15:45:35 PM
+ */
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, maxerr: 50 */
+/*global define*/
+define(function (require, exports, module) {
```

Comment:
Files are generally mixed tabs/spaces. For consistency, we should stick to 4 spaces.